Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -190,7 +205,10 @@ static void launchWithExtensionKit(ProcessLauncher& processLauncher, ProcessLaun
             [process invalidate];
             return;
         }
-        callOnMainRunLoop([weakProcessLauncher = weakProcessLauncher, name = name, process = RetainPtr<_SEExtensionProcess>(process)] {
+
+        RefPtr<LaunchGrant> launchGrant = adoptRef(*new LaunchGrant(process));
```

Comment:
`RefPtr<LaunchGrant>` -> `Ref`. Also, you shouldn't be doing this manually. Please add a static `create()` function to LaunchGrant and use that, like we always do (and for good reason). Also, please make the constructor private.

Reply:
Fixed!